docs/guides/ecosystem/docker.md

```diff
@@ -39,7 +39,7 @@ COPY . .
 # [optional] tests & build
 ENV NODE_ENV=production
 RUN bun test
-RUN bun run build
+RUN bun run build --target=bun
 
 # copy production dependencies and source code into final image
 FROM base AS release
```

Comment:
honestly I wonder how often you actually need to copy the dependencies as `bun build` should bundle everything needed into the one JS file

but then it is running the source TS file, so what exactly is the point on building?